psalm-baseline.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.19.0@a2ad69ae4f5ab1f7d225a8dc4e2ec2d9415ed599">
+  <file src="src/Language/Lexer.php">
+    <InvalidParamDefault occurrences="1">
+      <code>ParserOptions</code>
+    </InvalidParamDefault>
+    <InvalidTypeImport occurrences="1">
+      <code>ParserOptions from Parser</code>
+    </InvalidTypeImport>
+  </file>
+  <file src="src/Language/Parser.php">
+    <UndefinedDocblockClass occurrences="5">
+      <code>ParserOptions</code>
+      <code>ParserOptions</code>
+      <code>ParserOptions</code>
+      <code>ParserOptions</code>
+      <code>array{string, ParserOptions}</code>
+    </UndefinedDocblockClass>
+  </file>
+  <file src="src/Language/Visitor.php">
+    <EmptyArrayAccess occurrences="1">
+      <code>$path[count($path) - 1]</code>
+    </EmptyArrayAccess>
+  </file>
+  <file src="src/Type/Definition/FieldDefinition.php">
```

Comment:
I initially hoped to fix this especially but there's some crazy issue I'm not even able to isolate now.

Reply:
The type declarations in there are a little crazy - the full version was too crazy for PHPStan to handle 😆

Reply:
I'll definitely revisit after https://github.com/vimeo/psalm/issues/6928 https://github.com/vimeo/psalm/issues/7527 are solved

Reply:
Yeah. I did not feel like actually messing around in there tool beyond the PR I attempted. Using regex for parsing of something as complex as a type definition seems like a slippery slope.

Reply:
> Using regex for parsing of something as complex as a type definition seems like a slippery slope.

exactly, already encountered so many issues caused by that :/

Reminds me of https://stackoverflow.com/a/1732454/519333